be/src/exec/workgroup/io_dispatcher.cpp

```diff
@@ -0,0 +1,39 @@
+#include "io_dispatcher.h"
+
+namespace starrocks::workgroup {
+
+IoDispatcher::IoDispatcher(std::unique_ptr<ThreadPool> thread_pool) : _thread_pool(std::move(thread_pool)) {}
```

Comment:
use std::move or const std::unique_ptr<ThreadPool>,
std::unique_ptr have copy ctor and assignment operator.